lib/libcxxrt/Version.map

```diff
@@ -315,6 +315,14 @@ CXXRT_1.0 {
     __cxa_increment_exception_refcount;
     __cxa_rethrow_primary_exception;
 
+    /*
+     * HACK: also include the type_info values capabilities (U3cap) mangling
```

Comment:
I can't quite parse this sentence for whatever reason, does this mean "include type_info for capability types but in a mangled form"?  E.g. "typeinfo for void * __capability" or does it mean type_info for `uintcap_t` and `intcap_t`?  Note how earlier in the file we use a comment for mangled entries to give the unmangled name a move which might help?

Reply:
We are not exporting type infos for foo*, foo* const, etc. since the purecap version will contain a U3cap.

I'll rewrite the comment and see if I can come up with a fix in lld.

I also noticed that the type info for char8_t is missing but I'll submit that as an upstream review.